enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -676,42 +677,9 @@ flowchart TD
         ProxyDeployment
         LBService
     end
-    Gateway[Gateway] -.-> ProxyDeployment(Proxy Deployment)
-    Gateway[Gateway] -.-> LBService(LB Service)
     xRoute[xRoute] --> Gateway[Gateway]
 ```
 
-#### Automated and Manual Gateway Deployments
```

Comment:
Remove the reference to manual gateway deployments.

Reply:
Following up in this comment on https://github.com/openshift/enhancements/pull/1771/files#r2015033574

Reply:
I feel really odd about leaving any of this section, because it's rooted in a comparison of modes which aren't relevant anymore... however when I get back in tomorrow morning I'll try and take a fresh look at it :+1:

Reply:
Keeping the section on how automated deployments work is helpful.

Reply:
> Keeping the section on how automated deployments work is helpful.

It may have been helpful to hear specifically why you feel this is helpful, as that would have helped me make sure I was capturing what you felt was missing. I came back to it today and still wasn't _certain_, but reading it back over, I decided that maybe a little bit of "how Gateway resources work" was the helpful thing here. As such I had a go at a new "Automated Gateway Deployments" section. I moved it above the topologies section, because I realized it was potentially helpful context prior to reading about the topologies. LMKWYT, or if you still feel there's more needed here?

Reply:
Looks good.  I added a couple of comments for clarification.